src/garage/np/algos/batch_polopt.py

```diff
@@ -133,14 +136,13 @@ def process_samples(self, itr, paths):
 
         tabular.record('Iteration', itr)
```

Comment:
I think Itr, AverageDiscountedReturn, EpisodeRewardMean, and Num(Evaluation)Trajs should all go in evaluate_performance